.gitattributes

```diff
@@ -1 +1,4 @@
 * text=auto
+
+# force LF line ending for these files
+*.sh text eol=lf
```

Comment:
I think you can put the `eol=lf` directive above for all files so we don't get caught off-guard again.